CHANGELOG.md

```diff
@@ -38,6 +38,7 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 - Fixes the border and padding from the `fieldset` HTML element by removing it.
 - Fixes alignment for long labels in the `Checkbox` and `Radio` components.
 - Fixes the line height for long labels in the `Toggle` component.
+- Fixes bug where the Next button in `Pagination` would navigate to the previous page.
```

Comment:
This should be moved to a new `### Fixes` section under the `## Prerelease` section (which will be for 0.25.13).